src/service/db/alerts/templates.rs

```diff
@@ -123,28 +149,27 @@ pub async fn _watch() -> Result<(), anyhow::Error> {
         };
         match ev {
             db::Event::Put(ev) => {
-                let item_key = ev.key.strip_prefix(key).unwrap();
-                let item_value: Template = if config::get_config().common.meta_store_external {
-                    match db::get(&ev.key).await {
-                        Ok(val) => match json::from_slice(&val) {
-                            Ok(val) => val,
-                            Err(e) => {
-                                log::error!("Error getting value: {}", e);
-                                continue;
-                            }
-                        },
-                        Err(e) => {
-                            log::error!("Error getting value: {}", e);
-                            continue;
-                        }
+                let item_key = ev.key.strip_prefix(TEMPLATE_WATCHER_PREFIX).unwrap();
+                let keys = item_key.split('/').collect::<Vec<_>>();
+                if keys.len() != 2 {
+                    log::error!("Error event key not formatted correctly. Should be org_id/name, but got {}", item_key);
+                    continue;
+                }
```

Comment:
Nitpick: 
It might be nice to factor key generation and parsing into their own helper methods. When I was refactoring `/service/db/alerts.rs` I sometimes had a little trouble understanding exactly what was supposed to be happening in the key parsing logic so having this type of logic isolated to its own function(s) could help with readability/understandability.